docs/metrics/README.md

```diff
@@ -0,0 +1,4 @@
+# Getting Started with OpenTelemetry .NET Metrics in 5 Minutes
+
+* [Getting started with Counter](.\getting-started\README.md)
```

Comment:
might be better to make getting-started with counter as part of the main readme, and everything else under a separate directory "more instruments" or something. Listing all 4 (or 6) instruments in the main readme.md might mean it'd be more than "5 minutes" to really get started  :)